ui/app/styles/core/box.scss

```diff
@@ -4,7 +4,11 @@
  */
 
 .box {
+  background-color: $white;
   box-shadow: 0 0 0 1px rgba($grey-dark, 0.3);
+  color: $ui-gray-900;
+  display: block;
+  padding: 1.25rem;
```

Comment:
That brings up an interesting question for the future since to me `$size-5` should be a variable that can be used for anything, not necessarily text. If we wanted text specific sizing variables we should think about something like `$text-size-5`.